src/providers/utils.ts

```diff
@@ -0,0 +1,9 @@
+import BigNumber from "bignumber.js"
+
+export function formatNumber(value, decimals?: number) {
```

Comment:
could alternatively just default decimals arg to 18 and get rid of the else branch I think